.travis.yml

```diff
@@ -31,10 +31,9 @@ before_install:
 # Install packages
 install:
   - conda install --yes python=$TRAVIS_PYTHON_VERSION --file requirements.txt
```

Comment:
if wget is now in `pip` we should be able to install the `requirements.txt` via `pip install -r requirements.txt` too.